projects/farmworker-wallet-os.md

```diff
@@ -0,0 +1,120 @@
+
+# Current Code of Conduct
+[Farmworker Wallet OS Community Code of Conduct](https://github.com/Entidad/.github/blob/main/CODE_OF_CONDUCT.md)
+
+# TAC Sponsor
+_Include the name of a sponsor from the TAC, if identified (a sponsor helps mentor projects)_
+
+# Project License
+Because of the social impact origins of the Farmworker Wallet OS project, it is important to the initial contributors that source code not be used in ways that could bring harm to underserved communities.  As such, we propose to use the [Hippocratic License](https://firstdonoharm.dev/), Version 3.0, October 2021.  This license was created by the [Organization for Ethical Source](https://ethicalsource.dev/) in partnership with [Corporate Accountability Lab](https://corpaccountabilitylab.org/). The Hippocratic License is largely based on the Apache 2.0 license, with the addition of ethical standards. [Farmworker Wallet OS project proposed license](https://firstdonoharm.dev/version/3/0/eco-extr-law-media-mil-soc-sv-usta.html).
+
+# Source Control
+Github
+
+# Issue Tracker
+Github
+
+# External Dependencies
+1. [Mendix Studio Pro v9.24.3](https://marketplace.mendix.com/link/studiopro/) (integrated developer environment) 
+2. [Eclipse Temurin JDK 11 (x64)](https://adoptium.net/temurin/releases/?version=11)
+3. [Mendix Native Template v7.0.1](https://docs.mendix.com/refguide9/mobile/distributing-mobile-apps/building-native-apps/native-template/)
+4. [React Native v0.70.7](https://reactnative.dev/versions)
+5. Development instance of an [Aries Mediator Service](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0046-mediators-and-relays) 
+6. Development instance of an [Aries Cloud Agent](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0004-agents) 
+
+# Release Methodology
+_If the project already exists, please provide details on the release methodology and mechanics._
+
+# Initial Maintainers
+* [Ockert van Schalkwyk](https://github.com/skullquake)
+* [Jorge Flores](https://github.com/jorgefl0)
+
+# Proposed Project Governance
```

Comment:
How would others contribute to and become maintainers of the project?

Reply:
There is a lot of complexity with emerging web3 and digital trust technologies and we recognize that we need collaborate with likeminded developers in order to keep up with the rapidly evolving standards and protocols. We welcome other contributors and maintainers to join the project. 

Low-code app platforms have gained a lot of popularity in recent years and adoption will only continue to grow. We are hopeful that this open source project under the umbrella of the OWF would help attract other developers looking to make in impact with digital trust technologies with a significantly lower barrier to entry enabled by enterprise-grade low-code platforms like Mendix. Non-profits organizations, governments, and public service providers will be able to build privacy-preserving apps without braking the bank.

Reply:
I am happy to hear that you are looking for additional contributors and maintainers. It would be good to document how you would expect people to move through the funnel of user -> contributor -> maintainer.

Reply:
@tkuhrt This sounds like something that should be covered in a lifecycle plan. Can you clarify if this documentation is required for the Labs stage? We may have to reconsider applying for the [Growth Stage](https://tac.openwallet.foundation/governance/project-lifecycle/#growth-stage) if plans for contributor/maintainer participation need to be formalized.

Reply:
That is a good point. Let's talk about the expectations during the TAC call and be sure that everyone is on the same page.